yarn.lock

```diff
@@ -2348,6 +2358,25 @@ __metadata:
   languageName: node
   linkType: hard
 
+"apify-client@npm:^2.13.0":
```

Comment:
please deduplicate the lockfile so we dont have two installations of the client

(just run `yarn dedupe`)